SConstruct

```diff
@@ -310,6 +310,15 @@ else:
     )
     env.Append(CFLAGS=["-std=c11"])
 
+# At this time the only version of mingw that runs on arm64 windows is aarch64-w64-mingw32.
+# These settings should take effect in a msys2 clangarm64 environment.
+if env["platform"] == "windows" and env["arch"] == "arm64" and env.get("use_mingw", False):
```

Comment:
> env.get("use_mingw", False)

This condition probably should be the opposite, MinGW build (regardless if it's standalone or MSYS2 environment) should be run with `use_mingw=yes`.

Reply:
I updated this to only check for use_mingw. Ran it both ways on my desktop, it happily built with or without any issues. please let me know if this is not what you suggested.

Thanks!